db/migrate/20210312191925_add_username_tsvector_index_to_users.rb

```diff
@@ -0,0 +1,24 @@
+class AddUsernameTsvectorIndexToUsers < ActiveRecord::Migration[6.0]
+  disable_ddl_transaction!
+
+  INDEX = "to_tsvector('simple'::regconfig, COALESCE((username)::text, ''::text))"
+  private_constant :INDEX
+
+  def up
+    return if index_exists?(:users, INDEX, name: "index_usernames_as_tsvector")
```

Comment:
Can't wait to upgrade to Rails 6.1 and gain access to [the more native helpers](https://blog.saeloun.com/2020/05/11/if-not-exists-for-index.html) 🎉 .